istio-proxy/istio-proxy.spec

```diff
@@ -51,14 +53,32 @@ BuildRequires:  libatomic
 BuildRequires:  platform-python-devel
 
 Source0:        proxy-%{git_commit}.tar.gz
+Source10:       gn-%{gn_hash}.tar.gz
+Patch0:         0001-Preserve-LWS-from-the-middle-of-HTTP1-header-values.patch
+Patch10:        0002-http-header-map-security-fixes-for-duplicate-headers.patch
+
+ExclusiveArch:  x86_64
 
 %description
 The Istio Proxy is a microservice proxy that can be used on the client and server side, and forms a microservice mesh. The Proxy supports a large number of features.
 
 %prep
 %setup -q -n proxy-%{git_commit}
+%setup -q -D -T -n proxy-%{git_commit} -a 10
+pushd maistra/vendor/envoy
+%patch0 -p1
```

Comment:
Why are we patching it here instead of applying the changes in the repo directly?